client/src/components/pages/meditation.tsx

```diff
@@ -1,168 +1,159 @@
+      { title: "Feel Empowered", content: "You are full of energy and confidence. Embrace the positivity within you." },
+    ],
+    "stress-relief": [
+      { title: "Breathe In Calm", content: "Take a deep breath, and as you exhale, feel the tension begin to melt away." },
+      { title: "Release the Stress", content: "Visualize the stress leaving your body with each exhale, drifting away like smoke." },
+      { title: "Relax Your Mind", content: "Let your thoughts become clear and calm. Focus only on the peaceful feeling you are cultivating." },
+      { title: "Let Go of Worry", content: "Release any worries or anxieties. Trust that everything is unfolding as it should." },
+      { title: "Final Relaxation", content: "Feel yourself deeply relaxed, free from stress, and at peace." },
+    ],
+    "self-compassion": [
+      { title: "Be Kind to Yourself", content: "Start by acknowledging your efforts. You are doing your best, and that is enough." },
+      { title: "Forgive Yourself", content: "Let go of any self-criticism. Offer yourself forgiveness and kindness." },
+      { title: "Accept Yourself", content: "Embrace yourself as you are, with all your strengths and imperfections." },
+      { title: "Show Yourself Love", content: "Imagine wrapping yourself in a blanket of love and compassion." },
+      { title: "You Are Enough", content: "You are worthy of love and kindness. Treat yourself with the compassion you deserve." },
+    ],
   };
 
-  const handleSubmitMood = async () => {
-    if (!mood || !userMessage) {
-      return alert("Please select a mood and provide a message.");
-    }
-  
-    setIsSubmitting(true);
-    try {
-      await axios.post("http://localhost:8080/journal", {
-        mood,
-        entry: userMessage,
-      });
-      alert("Your mood has been saved!");
-      
-      // Reset mood and userMessage after submitting
-      setMood("");
-      setUserMessage("");
-      
-      setStep((prevStep) => Math.min(prevStep + 1, meditationSteps.length - 1));
-    } catch (error) {
-      console.error("Error submitting mood:", error);
-      alert("Failed to submit mood. Please try again.");
-    } finally {
-      setIsSubmitting(false);
+  const themeBackgrounds: Record<MeditationTheme, string> = {
+    "full-body-relaxation": waterImage,
+    "believe-in-yourself": forestImage,
+    "time-to-sleep": nightImage,
+    "mindful-breathing": natureImage,
+    "positive-energy-boost": sunriseImage,
+    "stress-relief": calmImage,
+    "self-compassion": serenityImage,
+  };
+
+  const handleNavigation = (direction: "next" | "prev") => {
+    if (theme && meditationThemes[theme]) {
+      if (direction === "next") {
+        if (step < meditationThemes[theme].length) {
+          setStep((prevStep) => prevStep + 1);
+        }
+      } else if (direction === "prev") {
+        if (step > 0) {
+          setStep((prevStep) => prevStep - 1);
+        }
+      }
```

Comment:
clicking on the previous button on the 0th step probably should bring the user back to the main meditation page, currently it doesn't do anything

![image](https://github.com/user-attachments/assets/72d1d0ae-bb85-4740-8b35-94b9e31aeb9b)

Reply:
fixed!